### ISSUE TITLE
R 'sortable' HTML widgets are not interactive in Positron

### ISSUE THREAD

**Issue by jmcphers** [posit-dev/positron#2145]
## Positron Version:

```
Positron Version: 2024.01.0 (Universal) build 223
Code - OSS Version: 1.85.0
Commit: 20518f2f1097ce4125c43a3b36b17a854fdbb39a
Date: 2024-01-26T03:12:36.174Z
Electron: 25.9.7
ElectronBuildId: undefined
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 23.2.0
```

## Steps to reproduce the issue:

Run this R code:

```r
library(sortable)

rank_list(
    text = "You can drag, drop and re-order these items:",
    labels = c("one", "two", "three", "four", "five"),
    input_id = "example_2"
  )
```

A Sortable widget appears in the Plots pane, but it doesn't work (you can't actually sort anything).

<img width="597" alt="image" src="https://github.com/posit-dev/positron/assets/470418/60d888fc-6ef3-42b5-906a-cf7d0028b37c">

## What did you expect to happen?

You should get an interactive widget with styling, like RStudio:

<img width="687" alt="image" src="https://github.com/posit-dev/positron/assets/470418/abdaf033-d651-4bd6-9aee-a4128ae0752f">

## Were there any error messages in the output or Developer Tools console?

Yes, and they are pretty indicative of the error:

```
index.js:11     GET vscode-webview://0qpdnnpc99d1857m1q4dnk35r8np48qks1hmmcp95hj9th1dumv1/lib/sortable-rank-list-0.5.0/rank_list.css net::ERR_ACCESS_DENIED
index.js:11     GET vscode-webview://0qpdnnpc99d1857m1q4dnk35r8np48qks1hmmcp95hj9th1dumv1/lib/htmlwidgets-1.6.4/htmlwidgets.js net::ERR_ACCESS_DENIED
...
```

It looks like the mechanism that allows HTML widget resources to load into the Plots pane isn't work for this widget for some reason.

**Comment by jmcphers** [comment]
Should be resolved in #4151.

<img width="767" alt="image" src="https://github.com/user-attachments/assets/25b5ad2a-f54d-4b82-95b5-ccc73c6b4673">

Note that while the widget does render correctly and you can drag, you can't drop. I think this has to do with eventing in the overlay webview; dropping does work if you pop it out into a regular browser.

**Comment by testlabauto** [comment]
### Verified Fixed
    Positron Version(s) : 2024.07.0-107
    OS Version          : OSX

### Test scenario(s) 

library(sortable)

rank_list(
    text = "You can drag, drop and re-order these items:",
    labels = c("one", "two", "three", "four", "five"),
    input_id = "example_2"
  )

As @jmcphers mentioned, you can drag but not drop unless you pop out to a regular browser.

Link(s) to TestRail test cases run or created: 
N/A
